runserver.py

```diff
@@ -8,4 +8,4 @@
     port = int(sys.argv[1])
 except (IndexError, ValueError):
     port = 5000
-app.run('0.0.0.0', port=port)
+app.run('0.0.0.0', port=port, debug=True)
```

Comment:
Remove this. `DEBUG = True` should be specified in `instance/development.py`. We need to disable debug mode sometimes to test some functionality.